docs_user/modules/proc_adopting-the-object-storage-service.adoc

```diff
@@ -90,13 +89,13 @@ spec:
 
 .Verification
 
-* Inspect the resulting {object_storage} service pods:
```

Comment:
does {object_storage} variable already contain the word service?

Reply:
Yes.